server/controllers/system/csv-import.php

```diff
@@ -42,17 +42,17 @@ public function handler() {
         $file = fopen($fileUploader->getFullFilePath(),'r');
         $errors = [];
 
-        while(!feof($file)) {
-            $userList = fgetcsv($file);
+        $user = fgetcsv($file);
 
-            Controller::setDataRequester(function ($key) use ($userList) {
+        while(!feof($file)) {
+            Controller::setDataRequester(function ($key) use ($user) {
                 switch ($key) {
                     case 'email':
-                        return $userList[0];
+                        return $user[0];
```

Comment:
The function `fgetcsv` can return an array, but it also can return `false` if the read failed. It would be better to check the `fgetcsv` output every time to make sure it's not `false`, and do something if it is.